### ISSUE TITLE
Cannot import name `Utc`

### ISSUE THREAD

**Issue by Natim** [Kinto/kinto#1301]
While trying to debug #1299 I encountered the following error:
```
$ make serve
...
~/.virtualenvs/test/bin/kinto migrate --ini config/kinto.ini
Traceback (most recent call last):
  File "~/.virtualenvs/test/bin/kinto", line 11, in <module>
    load_entry_point('kinto', 'console_scripts', 'kinto')()
  File "~/.virtualenvs/test/lib/python3.5/site-packages/pkg_resources/__init__.py", line 560, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "~/.virtualenvs/test/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2648, in load_entry_point
    return ep.load()
  File "~/.virtualenvs/test/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2302, in load
    return self.resolve()
  File "~/.virtualenvs/test/lib/python3.5/site-packages/pkg_resources/__init__.py", line 2308, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "~/mozilla/kinto/kinto/__init__.py", line 4, in <module>
    import kinto.core
  File "~/mozilla/kinto/kinto/core/__init__.py", line 10, in <module>
    from kinto.core import errors
  File "~/mozilla/kinto/kinto/core/errors.py", line 1, in <module>
    import colander
  File "~/.virtualenvs/test/lib/python3.5/site-packages/colander/__init__.py", line 22, in <module>
    from . import iso8601
  File "~/.virtualenvs/test/lib/python3.5/site-packages/colander/iso8601.py", line 3, in <module>
    from iso8601.iso8601 import (parse_date, ParseError, Utc, FixedOffset, UTC, ZERO, ISO8601_REGEX)
ImportError: cannot import name 'Utc'
Makefile:87 : la recette pour la cible « migrate » a échouée
make: *** [migrate] Erreur 1
```

**Comment by Natim** [comment]
Refs https://github.com/Pylons/colander/issues/294

**Comment by Natim** [comment]
Refs https://bitbucket.org/micktwomey/pyiso8601/issues/27/possible-api-contract-break-in-0112-utc

**Comment by Natim** [comment]
I keep it open to remember to follow up on this.

**Comment by Natim** [comment]
Was fixed with colander 1.4.0
